<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-RCMCRH0" failures="1" tests="7" name="Amazon Functional Testing" time="60.257" errors="0" timestamp="2021-01-05T10:23:52 IST">
  <testcase classname="com.qa.testscripts.Amazon_SearchMultipleItems" name="CheckTitle" time="0.542">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Welcome to Amazon] but found [Online Shopping site in India: Shop Online for Mobiles, Books, Watches, Shoes and More - Amazon.in]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Welcome to Amazon] but found [Online Shopping site in India: Shop Online for Mobiles, Books, Watches, Shoes and More - Amazon.in]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.qa.testscripts.Amazon_SearchMultipleItems.CheckTitle(Amazon_SearchMultipleItems.java:100)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
    </failure>
  </testcase> <!-- CheckTitle -->
  <testcase classname="com.qa.testscripts.Amazon_SearchMultipleItems" name="CheckFirstSelectedOptions" time="2.214"/>
  <testcase classname="com.qa.testscripts.Amazon_SearchMultipleItems" name="getAllLinks" time="18.59"/>
  <testcase classname="com.qa.testscripts.Amazon_SearchMultipleItems" name="FooterLinks" time="5.405"/>
  <testcase classname="com.qa.testscripts.Amazon_SearchMultipleItems" name="SearchItem" time="11.94"/>
  <testcase classname="com.qa.testscripts.Amazon_SearchMultipleItems" name="CheckTitleLoaded" time="2.462"/>
  <testcase classname="com.qa.testscripts.Amazon_SearchMultipleItems" name="CheckItemsLoaded" time="1.687"/>
</testsuite> <!-- Amazon Functional Testing -->
